Runtimes/Overlay/Cxx/CMakeLists.txt

```
@@ -1,8 +1,12 @@
 
+include(CheckSymbolExists)
+check_symbol_exists(_LIBCPP_VERSION "verison" HAVE_LIBCPP_VERSION)
```

Is this line needed at all? I don't see its value being used

If this is needed, I suspect there may be a typo in the header name

```suggestion
check_symbol_exists(_LIBCPP_VERSION "version" HAVE_LIBCPP_VERSION)
```

---

Yes, its the same typo; I think that we should keep, it should be pretty cheap.